Grammatical adjustments of `page models` usage guide to resolve issue #9398 (#9399)
wagtail/wagtail: docs/topics/pages.md

# Page models

> In Wagtail, a Django model represents each page type (a.k.a. content type). All page models must inherit from the {class}`wagtail.models.Page` class.

Reviewer: I find the existing content much clearer.

```suggestion
Each page type (a.k.a. content type) in Wagtail is represented by a Django model. All page models must inherit from the {class}`wagtail.models.Page` class.
```

### Page descriptions

> You can add helpful description text to every Wagtail Page, similar to a `help_text` model attribute. By adding `page_description` to your Page model, you'll be adding a short description that can be seen when you create a new page, edit an existing page or when you're prompted to select a child page type.

Reviewer: ```suggestion
With every Wagtail Page you are able to add a helpful description text, similar to a `help_text` model attribute. By adding `page_description` to your Page model you'll be adding a short description that can be seen when you create a new page, edit an existing page or when you're prompted to select a child page type.
```

#### Obtaining URLs for page instances

You can call the `Page.get_url(request)` method whenever you need a page URL. It defaults to returning local URLs (not including the protocol or domain) if it determines that the page is on the current site (via the hostname in `request`); otherwise, it would return a full URL including the protocol and domain. Whenever possible, you should include the optional `request` argument to enable per-request caching of site-level URL information and facilitate the generation of local URLs.

A common use case for `get_url(request)` is in any custom template tag your project may include for generating navigation menus. When writing such a custom template tag, ensure that it includes `takes_context=True` and use `context.get('request')` to safely pass the
request or `None` if no request exists in the context.

For more information, please see {meth}`wagtail.models.Page.get_url`.

> If you need the full URL (includes the protocol and domain), you can use `Page.get_full_url(request)` method. To enable per-request caching of site-level URLs, the optional `request` argument should be included whenever possible.

Reviewer: ```suggestion
To retrieve the full URL (including the protocol and domain), use `Page.get_full_url(request)`. Whenever possible, the optional `request` argument should be included to enable per-request caching of site-level URL information.
```
